core/src/validator.rs

```diff
@@ -545,13 +575,19 @@ impl Validator {
                 ));
             }
 
-            let (confirmed_bank_sender, confirmed_bank_receiver) = unbounded();
-
             let accountsdb_repl_service = config.accountsdb_repl_service_config.as_ref().map(|accountsdb_repl_service_config| {
+                let (confirmed_bank_sender, confirmed_bank_receiver) = unbounded();
+                confirmed_bank_senders.push(confirmed_bank_sender);
```

Comment:
I think the "confirmed_bank" prefix is no longer accurate, since the plugin service is using the frozen notification separately.
How about "bank_notification_sender/bank_notification_receiver", etc?

Reply:
Oh, I see you did make these kinds of nomenclature changes in optimistically_confirmed_bank_tracker.rs.
Let's do that everywhere!

Reply:
Done